spec/views/hyrax/dashboard/collections/show.html.erb_spec.rb

```diff
@@ -69,7 +68,7 @@
 
     context 'and only works are in search results' do
       before do
-        assign(:subcollection_count, 0)
+        # assign(:subcollection_count, 0)
```

Comment:
Is this kruft now that things are working?  Looks like it needs to be removed.

Reply:
Yes, I missed it, removing it now.